Test.txt

```diff
@@ -1 +1,3 @@
-This is a test.
\ No newline at end of file
+This is how you change a file.
```

Comment:
Testing reply.

---
In reply to: [58288391](https://github.com/kivancmuslu/CFTest/pull/1#discussion_r58288391) [](ancestors = 58288391)